lib/JsonNode.cpp

```diff
@@ -991,7 +999,16 @@ bool JsonUtils::parseBonus(const JsonNode &ability, Bonus *b)
 
 	value = &ability["propagator"];
 	if (!value->isNull())
-		b->propagator = parseByMap(bonusPropagatorMap, value, "propagator type ");
+	{
+		//ALL_CREATURES old propagator compatibility
+		if(value->String() == "ALL_CREATURES") 
+		{
+			b->addLimiter(std::make_shared<CreatureLevelLimiter>());

```

Comment:
perhaps add warning / conversion information for mods?

Reply:
@IvanSavenko for now,limiters and propagators cannot be handled by converter. Will add a warning.

Reply:
warning done